static/js/pages/institutionSelection.js

```diff
@@ -302,6 +302,8 @@ function linkClickHandler(container, institution, type, searching, e) {
         console.error(e);
     });
 
+    document.getElementById("footer").innerHTML = "";
```

Comment:
I guess this clears and creates proceed button again. Where is the place button is created on click?